hips/tiles/description.py

```diff
@@ -1,26 +1,38 @@
 # Licensed under a 3-clause BSD style license - see LICENSE.rst
 from collections import OrderedDict
 
+__doctest_skip__ = ['HipsSurveyProperties']
+
 __all__ = [
-    'HipsDescription',
+    'HipsSurveyProperties',
 ]
 
 
-class HipsDescription:
+class HipsSurveyProperties:
     """HiPS properties container.
 
     Parameters
     ----------
     properties : `~collections.OrderedDict`
-        HiPS description properties
+        HiPS survey properties
+
+    Examples
+    --------
+    ::
+
+        >>> from astropy.utils.data import get_pkg_data_filename
```

Comment:
You're still using `get_pkg_data_filename` from a docs example here. As discussed previously, this won't work. Please change to a working example where you instead fetch a file from Github or CDS.